docs/changelog.rst

```diff
@@ -19,6 +19,7 @@ ChangeLog
 *Removed:*
 
 - Drop support for Django 4.0
+- Drop support for Django 4.1
```

Comment:
I'd rephrase this as "Stop testing for Django 4.1 compatibility".
We're not actively dropping that support.

Reply:
We’re removing the PyPI marker and no longer care for compatibility with Django 4.1, so I think drop support is accurate, although we’re not removing django 4.1 specific code?

Reply:
Well, when I read the changelog of a third-party package and see there "Drop support for Python 2", I understand that the code is likely to break with Python 2, for instance :)

Reply:
Code could break for Django 4.1 and we wouldn’t know, or care. But enough discussion on this topic, I updated to a more precise phrasing.

Reply:
When I see the `"Drop support for X"` in other projects I tend to view it as "use at your own risk", it may or may not work... Just my $0.02.

What you've got is fine too.